.github/workflows/main.yml

```diff
@@ -74,20 +74,28 @@ jobs:
     - name: Run Ruby tests
       run: bundle exec rake compile_no_debug
 
-  build-jruby:
+  build-java:
     runs-on: ubuntu-latest
     env:
       JRUBY_OPTS: "--dev"
     steps:
     - uses: actions/checkout@v3
+    - name: Set up CRuby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.2
+        bundler-cache: true
+    - name: Serialize fixtures on CRuby
+      run: bundle && bundle exec rake compile test:serialize_fixtures
```

Comment:
You shouldn't need `bundle &&` here because setup ruby does that.

Reply:
Oh true and I should know that ahah (I just copy/pasted the command I used locally)

Reply:
Ah it's because I tried to use the system Ruby on GHA but that didn't work (doesn't have bundler)